package/rancher-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# first arg is `-flag` or `--some-flag`
+if [ "${1:0:1}" = '-' ]; then
```

Comment:
basically so that one can start a container both by specifying the binary path and without it:

`docker run rancher/external-dns /usr/bin/external-dns -debug -provider=cloudflare`
or
`docker run rancher/external-dns -debug -provider=cloudflare`

the templates just passes the flags in the command for example: https://github.com/rancher/rancher-catalog/blob/master/infra-templates/route53/11/docker-compose.yml#L3